src/QuickMarcEditor/QuickMarcEditor.test.js

```diff
@@ -214,4 +215,25 @@ describe('Given Quick Marc Editor', () => {
       expect(getByText('ui-quick-marc.authority-record.edit.title')).toBeDefined();
     });
   });
+
+  describe('when marc record is of type HOLDINGS', () => {
```

Comment:
There is a test suite `when marc record is of type HOLDINGS` already, it is a bit higher. It's better to move your test case there.